examples/LoRaReceiverCallback/LoRaReceiverCallback.ino

```diff
@@ -1,43 +1,45 @@
-#include <SPI.h>
-#include <LoRa.h>
-
-#ifdef ARDUINO_SAMD_MKRWAN1300
-#error "This example is not compatible with the Arduino MKR WAN 1300 board!"
-#endif
-
-void setup() {
-  Serial.begin(9600);
-  while (!Serial);
-
-  Serial.println("LoRa Receiver Callback");
-
-  if (!LoRa.begin(915E6)) {
-    Serial.println("Starting LoRa failed!");
-    while (1);
-  }
-
-  // register the receive callback
-  LoRa.onReceive(onReceive);
-
-  // put the radio into receive mode
-  LoRa.receive();
-}
-
-void loop() {
-  // do nothing
-}
-
-void onReceive(int packetSize) {
-  // received a packet
-  Serial.print("Received packet '");
-
-  // read packet
-  for (int i = 0; i < packetSize; i++) {
-    Serial.print((char)LoRa.read());
-  }
-
-  // print RSSI of packet
-  Serial.print("' with RSSI ");
-  Serial.println(LoRa.packetRssi());
-}
-
+#include <SPI.h>
+#include <LoRa.h>
+
+#ifdef ARDUINO_SAMD_MKRWAN1300
+#error "This example is not compatible with the Arduino MKR WAN 1300 board!"
+#endif
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  Serial.println("LoRa Receiver Callback");
+
+  if (!LoRa.begin(915E6)) {
+    Serial.println("Starting LoRa failed!");
+    while (1);
+  }
+
+  // set LNA gain (optional)
+  // LoRa.setGain(6); ranges from 0-6, default 0, see API docs

```

Comment:
Looking good!

Some minor suggestions on wording/formatting, how about something like:

```c
// Uncomment the next line to disable the default ADC gain, values between 1 - 6 are supported
// LoRa.setGain(6);
```

What I wrote above is probably not super clear ... would there be some particular use cases where manual gain is preferred ?

Reply:
Then we miss the case for LoRa.setGain(0) to set back the AGC?
For tracking via RSSI, I may want the gain to be fixed. So the AGC won't interfere with the gain and adjust the RSSI unexpected.

But for now, I see in short range around 10 meters, LoRa.setGain(6) will give much better RSSI.

How about the below?
// Uncomment the next line to disable the default ADC gain, values between 1 - 6 are supported
// Comment the next line or setGain(0) to enable the default AGC gain
// LoRa.setGain(6);

Reply:
Should be like this

// Uncomment the next line to disable the default AGC and set LNA gain, values between 1 - 6 are supported
// Comment the next line or setGain(0) to enable the default AGC
// LoRa.setGain(6);

Reply:
> Then we miss the case for LoRa.setGain(0) to set back the AGC?

but the default is AGC right?

> For tracking via RSSI, I may want the gain to be fixed. So the AGC won't interfere with the gain and adjust the RSSI unexpected.
> 
> But for now, I see in short range around 10 meters, LoRa.setGain(6) will give much better RSSI.

This would be good for the API.md description ...

> How about the below?
> // Uncomment the next line to disable the default ADC gain, values between 1 - 6 are supported
> // Comment the next line or setGain(0) to enable the default AGC gain

but if the line is commented out already this is confusing? Maybe we can keep it simple an link to the API.md file?

> // LoRa.setGain(6);

Reply:
Done.
https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/ac6d0a1d2276c45640a1ad7cb9ad9761df7baa3e

Reply:
Yah, by default is AGC.
Ok, so keep it simple like yours? The API has full explanation. 

// Uncomment the next line to disable the default ADC gain, values between 1 - 6 are supported
// LoRa.setGain(6);

Reply:
Ok, I put like your suggestion.
```
  // Uncomment the next line to disable the default AGC and set LNA gain, values between 1 - 6 are supported
  // LoRa.setGain(6);
```
https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/f287bed5e007103cece5ec7c1a84bd188032ddf2